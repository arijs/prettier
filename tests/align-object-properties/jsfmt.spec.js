run_spec(__dirname, { parser: "babylon", alignObjectProperties: true });
