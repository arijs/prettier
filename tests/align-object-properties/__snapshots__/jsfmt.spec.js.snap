// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`o.js 1`] = `
o = {
  [name]: 'value',
  2:3,
  "k-2":2,
  keyasdf:3
}

o = {
  [namelonglonglong]: 'value',
  2:3,
  "k-2":2,
  keyasdf:3
}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
o = {
  [name]  : "value",
  2       : 3,
  "k-2"   : 2,
  keyasdf : 3
};

o = {
  [namelonglonglong] : "value",
  2                  : 3,
  "k-2"              : 2,
  keyasdf            : 3
};

`;
